xcp_d/utils/bids.py

```diff
@@ -270,6 +270,16 @@ def collect_data(
                 queries["template_to_t1w_xform"]["from"] = space
                 break
 
+        if input_type in ("hcp", "dcan"):
+            temp_allowed_spaces = ["MNI152NLin6Asym"]
+            # HCP and DCAN files don't have nifti BOLD data, we will use the boldref
+            temp_bold_query["desc"] = None
+            temp_bold_query["suffix"] = "boldref"
+            temp_bold_query["space"] = "MNI152NLin6Asym"
+            queries["t1w_to_template_xform"]["to"] = "MNI152NLin2009cAsym"
+            queries["template_to_t1w_xform"]["from"] = "MNI152NLin2009cAsym"
+            nifti_bold_data = layout.get(**temp_bold_query)
```

Comment:
Just want to make a note that this is a hack that we should drop. I'm thinking we should probably search directly for transforms, since the BOLD file isn't actually used. That can go in a future PR though.

Reply:
Will make a note